### PR TITLE
Aws doc: troubleshooting section

### PR DESCRIPTION
**Pull request by sbrossie**

### CODE REVIEW

**Comment by sbrossie** [comment]
@pierre We cannot see my screenshots until this is merge because of the links `https://github.com/killbill/killbill-docs/raw/v3/userguide/...`

File: userguide/tutorials/aws.adoc [hunk 1]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+
+==== Stack Creation
+
+There are a couple of possible issues when starting the stack from CloudFormation (CFN) and so the goal of this section
+is to provide some tips debugging the issues, and knowledge about where to find various logs. Also keep in mind
+that starting the CFN stack will typically take in the order of 15-20', mostly because of the time it takes to setup the 
+RDS cluster, so be patient...
+
+
+Let's start with the https://console.amazonaws.cn/cloudformation[CloudFormation console]: The stack will have a status which will be `CREATE_IN_PROGRESS` for the duration of the initialization, and will then either transition to `CREATE_COMPLETE` or `CREATE_FAILED`. In case of `CREATE_FAILED`, try to locate from the `Resources` tab which resource fail to initialize properly.
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: Point to a US console instead of the China one (`.cn`).

File: userguide/tutorials/aws.adoc [hunk 2]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+The main possible issues can be summarized as:
+
+**Issues with the template**
+
+If you think there is an issue with the CFN template itself, please report any issue to `support@killbill.io`
+
+**AWS Errors**
+
+Typical Issues are:
+  * Insufficient IAM Permissions
```
Comment (pierre) [inline userguide/tutorials/aws.adoc:165]: You need a new line before the first `*` for the list to render properly.

File: userguide/tutorials/aws.adoc [hunk 3]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+  * Limit Exceeded
+  * Security Group Does Not Exist in VPC
+  * RDS Cluster failed to come up
+  * ...
+
+Make sure to check our `Pre-requisites` section to ensure you have setup a valid VPC with valid subnets. Fow AWS specific issues, please refer to the https://docs.amazonaws.cn/en_us/AWSCloudFormation/latest/UserGuide/troubleshooting.html[AWS troubleshooting documentation]
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: China link

File: userguide/tutorials/aws.adoc [hunk 4]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+3. Is the database schema correctly installed?
+4. Is the Kill Bill/Kaui server correctly started and listening to the port `8080`?
+5. Are Kill Bill/Kaui server accessible from the LB, respectively on port `80` and `9090`?
+6. Are there any errors, stack traces in our logs
+
+==== Practical Tips
+
+**SSH to EC2 Instances**
+
+In order to answer these questions, you will first need to be able to SHH to the EC2 instances:
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: `SSH`

File: userguide/tutorials/aws.adoc [hunk 5]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+
+From the EC2 dashboard, you can locate the instances by filtering on a prefix of the name of your stack, in my case `my-stack`,
+and as indicated below you will see the instances for KIll Bill server and KAUI. In the example below we see one of each:
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: K`i`ll

File: userguide/tutorials/aws.adoc [hunk 6]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ec2-instances.png[align=center]
+
+You can select one instance and and then from the description tag, you will have access to:
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: `and and`

File: userguide/tutorials/aws.adoc [hunk 7]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+1. Public DNS
+2. The security group
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/ec2-description.png[align=center]
+
+You will need to first click on the security group link to open the inbound port 22 required for SHH, as shown below:
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: `SSH`

File: userguide/tutorials/aws.adoc [hunk 8]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+
+image:https://github.com/killbill/killbill-docs/raw/v3/userguide/assets/aws/security-group.png[align=center]
+
+Then, you can issue the SHH command, by copying the Public DNS from the description tab:
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: `SSH`

File: userguide/tutorials/aws.adoc [hunk 9]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+/var/log/awslogs.logs
+/var/log/xray/xray.log
+```
+
+**Kill Bill/KAUI Server Logs**
+
+The Kill Bill/KAUI server logs are located under `/var/lib/tomcat/logs/`, with the main 2 interesting logs being:
+* `killbill.out`: All Kill Bill server logs, configured as `INFO` by default
+* `kaui.out`: All KAUI server logs, configured as `INFO` by default
+* `localhost_access_log...`: Access requests to the severs
```
Comment (pierre) [inline userguide/tutorials/aws.adoc]: `servers`

File: userguide/tutorials/aws.adoc [hunk 10]
```diff
@@ -138,6 +138,179 @@ You can log-in to Kaui by going to http://<LOAD_BALANCER_URL>:9090 (make sure yo
...
+
+The configuration of the logging (log rotation, log level, ...) can be found in `/var/lib/killbill/config/logback.xml`
```
Comment (pierre) [inline userguide/tutorials/aws.adoc:236]: Maybe specify that updating that file requires a Tomcat restart (and specify the `service` commands)?

**Review by pierre — APPROVED** [review]
Very nice! Just found a few typos...

**Comment by sbrossie** [comment]
@pierre Updated PR with https://github.com/killbill/killbill-docs/pull/89/commits/6a97643b0b8527a9b316fc80e9c3cd30356acff0

**Pull request merged**